tests/sqlparser_snowflake.rs

```diff
@@ -525,6 +525,307 @@ fn test_snowflake_single_line_tokenize() {
+                            option: ColumnOption::Identity(Identity::Autoincrement(
+                                IdentityProperty {
+                                    #[cfg(not(feature = "bigdecimal"))]
+                                    parameters: Some(IdentityFormat::FunctionCall(
+                                        IdentityParameters {
+                                            seed: Expr::Value(Value::Number(
+                                                "100".to_string(),
+                                                false
+                                            )),
+                                            increment: Expr::Value(Value::Number(
+                                                "1".to_string(),
+                                                false
+                                            )),
+                                        }
+                                    )),
+                                    #[cfg(feature = "bigdecimal")]
+                                    parameters: Some(IdentityFormat::FunctionCall(
+                                        IdentityParameters {
+                                            seed: Expr::Value(Value::Number(
+                                                bigdecimal::BigDecimal::from(100),
+                                                false,
+                                            )),
+                                            increment: Expr::Value(Value::Number(
+                                                bigdecimal::BigDecimal::from(1),
+                                                false,
+                                            )),
+                                        }
+                                    )),
+                                    order: Some(IdentityOrder::Noorder),
+                                }
+                            ))
+                        }]
+                    },
+                    ColumnDef {
+                        name: "c".into(),
+                        data_type: DataType::Int(None),
+                        collation: None,
+                        options: vec![ColumnOptionDef {
+                            name: None,
+                            option: ColumnOption::Identity(Identity::Identity(IdentityProperty {
+                                parameters: None,
+                                order: None,
+                            }))
+                        }]
+                    },
+                    ColumnDef {
+                        name: "d".into(),
+                        data_type: DataType::Int(None),
+                        collation: None,
+                        options: vec![ColumnOptionDef {
+                            name: None,
+                            option: ColumnOption::Identity(Identity::Identity(IdentityProperty {
+                                #[cfg(not(feature = "bigdecimal"))]
```

Comment:
Ah if the feature flags are for the value number there's a helper we can use e.g. `number("1")` to produce a number value

Reply:
Applied for tests of `snowflake` and `ms sql`